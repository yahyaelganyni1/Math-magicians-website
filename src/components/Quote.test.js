import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('expect quote to match snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
