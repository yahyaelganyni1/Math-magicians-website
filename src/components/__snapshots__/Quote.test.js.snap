// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match snapshot 1`] = `
<div
  className="quote"
>
  <h3>
    Without mathematics, thereâ€™s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  </h3>
</div>
`;
